packages/viewer/src/components/viewer-transform-widget/util.ts

```diff
@@ -199,6 +199,12 @@ function computeTranslation(
   return rotatedDelta.x + rotatedDelta.y + rotatedDelta.z;
 }
 
+/**
+ * This helper makes a translation matrix from the negated position
+ * @param rotationAxis
+ * @param current
+ * @returns
+ */
```

Comment:
nit: Could we update this description (and also potentially rename the `rotationAxis` param)? This method is actually applying the rotation at the current widget position, then translating it back to convert it to a world delta. The `rotationAxis` naming is a bit misleading, since it's actually a quaternion